### ISSUE TITLE
Table headers have not sufficient contrast

### ISSUE THREAD

**Issue by yatil** [w3c/web-roadmaps#231]
Table headers have a color combination of (#eeeeee / #0070cc) – 4.32:1.

#f3f3f3 / #0070cc has sufficient color contrast (although I’m unable to really see the difference)
